python/ptest/cases/gomspace_checkout_case.py

```diff
@@ -0,0 +1,126 @@
+# Gomspace test case. Gets cycle count purely for diagnostic purposes and logs
+# any other Gomspace state fields.
+from .base import SingleSatOnlyCase
+
+
+class GomspaceCheckoutCase(SingleSatOnlyCase):
+
+    def setup_case_singlesat(self, simulation):
+        simulation.flight_controller.write_state(
+            "pan.state", 9)  # Manual state
+        self.run_case_singlesat(simulation)
+        print("Gomspace cases finished.")
+
+    def run_case_singlesat(self, simulation):
+        simulation.cycle_no = simulation.flight_controller.read_state(
+            "pan.cycle_no")
+
+        def read_state(self, string_state):
+            return simulation.flight_controller.read_state(string_state)
+
+        def write_state(self, string_state, state_value):
+            simulation.flight_controller.write_state(string_state, state_value)
+            return read_state(self, string_state)
+
+        # readable fields
+
+        vboost = [int(read_state(self, "gomspace.vboost.output" + str(i)))
+                  for i in range(1, 4)]
+        for n in range(0, len(vboost)):
+            print("vboost" + str(n) + " is: " + str(vboost[n]) + " mV")
+
+        vbatt = int(read_state(self, "gomspace.vbatt"))
+        if vbatt < 6000 or vbatt > 8400:
+            print(
+                "Vbatt is out of expected range [6000, 8400] mV at: " + str(vbatt))
+
+        curin = [int(read_state(self, "gomspace.curin.output" + str(i)))
+                 for i in range(1, 4)]
+        for n in range(0, len(curin)):
+            print("curin" + str(n) + " is: " + str(curin[n]) + " mA")
+        cursun = int(read_state(self, "gomspace.cursun"))
+        print("cursun is: " + str(cursun) + " mA")
+        cursys = int(read_state(self, "gomspace.cursys"))
+        print("cursys is: " + str(cursys) + " mA")
+
+        curout = [int(read_state(self, "gomspace.curout.output" + str(i)))
+                  for i in range(1, 7)]
+        for n in range(0, len(curout)):
+            print("curout" + str(n) + " is: " + str(curout[n]) + " mA")
+
+        output = [bool(read_state(self, "gomspace.output.output" + str(i)))
+                  for i in range(1, 7)]
+
+        for n in range(0, len(output)):
+            out_n = output[n]
+            # checked umbilical for which outputs should be 0 and 1:
+            # 1-5 are all 5V, 6 is 3.3V
+            if out_n is False:
+                print("Output-" + n +
+                      " is not on: ")
```

Comment:
Judging by the colon here, I feel like you _might_ have wanted to add something more to the print statement and maybe forgot. It doesn't matter to me either way, but as a good citizen with a civic responsibility to my fellow (wo)man, I feel that I must bring this matter to your attention.